config/database.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'default' => env('DB_CONNECTION', 'pgsql'),
+    'default' => env('DB_CONNECTION', 'mysql'),
```

Comment:
Let us leave this as pgsql since it is what Heroku depends on, you can override it in your own .env file.

Reply:
Sorry I thought I had added config/database.php to the .gitignore. I use mysql in my dev environment so I changed this. I should have changed it back to pgsql. 
Let me fix this.